.github/workflows/ccpp.yml

```diff
@@ -301,3 +301,59 @@ jobs:
             else \
               exit 0; \
             fi
+  build-and-check-windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        include:
+          - { sys: mingw64, env: x86_64 }
+          - { sys: mingw32, env: i686 }
+    name: 'Windows ${{ matrix.env }} ${{ matrix.sys }}'
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: '${{ matrix.sys }} Install deps'
+        uses: msys2/setup-msys2@v2
+        with:
+          msystem: ${{matrix.sys}}
+          update: true
+          install: >-
+            git
+            make
+            dejagnu
+            mpc
+            mpfr
+            gmp
+            curl
+            sed
+            findutils
+            diffutils
+            flex
+            texinfo
+          pacboy: >-
+            toolchain:p
+            rust:p
+
+      - run: |
+          find . -name .cargo-checksum.json -exec sed -i.uncheck -e 's/"files":{[^}]*}/"files":{ }/' '{}' '+'
+
+
+      - name: Configure
+        run: |
+           mkdir -p gccrs-build;
+           cd gccrs-build;
+           ../configure \
+               --enable-languages=rust \
+               --disable-bootstrap \
+               --enable-multilib
```

Comment:
I don't think multilib is support on mingw-w64, We don't build mingw-w64-gcc with it.